plugin/distributed/distributed.go

```diff
@@ -95,11 +98,87 @@ const writeResultsAction = "writeResults"
 const requestResultKey = "results"
 
 // Just used for unmarshalling the results passed from osquery.
-type resultsStruct struct {
```

Comment:
If this is going to be an exported type, I think it should have a docstring like `ResultsStruct is...`